test/wallet/index.ts

```diff
@@ -165,12 +165,9 @@ describe('Wallet', function () {
         SigningPubKey: publicKey,
       }
       const wallet = new Wallet(publicKey, privateKey)
-      const signedTx: { signedTransaction: string; id: string } =
-        wallet.signTransaction(txJSON)
+      const signedTx: string = wallet.signTransaction(txJSON)
 
-      assert.hasAllKeys(signedTx, ['id', 'signedTransaction'])
-      assert.isString(signedTx.id)
-      assert.isString(signedTx.signedTransaction)
+      assert.isString(signedTx)
```

Comment:
Add a TODO at the least to make this a more rigorous check that actually checks the output of the signed transaction instead of just the type